### ISSUE TITLE
what is the loss of your speaker encoder model

### ISSUE THREAD

**Issue by hdmjdp** [Edresson/YourTTS#2]
Can you give the train log of the spk encoder? thanks a lot.

**Comment by Edresson** [comment]
Hi, we don't train the speaker encoder. We use the checkpoint publicly available from the authors of the [H/ASP model](https://arxiv.org/abs/2009.14153). You can find the training code and model details [here](https://github.com/clovaai/voxceleb_trainer#pretrained-models)
